docs/ssh.md

```diff
@@ -55,6 +55,10 @@ All other options are parsed but have no effect.  Notably, neither `Match` or
 
 `Include` is now supported.
 
+{{since('20240914-091937-5212cd4c:')}}
```

Comment:
I use nightly for this sort of thing; after I cut a release, I will replace these since-nightly markers with the version number of the newly tagged release.

```suggestion
{{since('nightly')}}
```

Reply:
Okay, I'll remember that for the future too.